src/Components/modal/Modal.css

```diff
@@ -0,0 +1,38 @@
+.modalDialog {
+  grid-area: pro;
+  justify-self: center;
+  margin-top: 5rem;
+  flex-wrap: wrap;
+}
+
+.modalDialog > div {

```

Comment:
I have a suggestion, instead of div, have one class like modal-content, so that if someone uses any other element than div, it will work by applying class to it.
And also you can keep consistency in class names like if you are using - between words, then use it throughout the doc, modalDialog should be modal-dialog